### PR TITLE
rename Moralis v1 web3

### PR DESCRIPTION
**Pull request by nikita12c**
---
name: 'Pull request'
about: A new pull request
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/web3ui/web3uikit/blob/master/CONTRIBUTE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description
rename Moralis web3 to web3parse and change related links in readme 

Related issue: #812

### Solution Description
renamed Moralis web3 to web3parse in all related components of section 1 and change related links in readme 
image

### CODE REVIEW

File: packages/web3/README.md [hunk 1]
```diff
@@ -101,4 +101,4 @@ The `<NFTBalance />` component allows you to display all the NFT's an address ow
 
 If you want to use this component wrap your app with `<MoralisProvider appId={MORALIS_APP_ID} serverUrl={MORALIS_SERVER_URL} ></MoralisProvider >`. Read more about moralis [here](https://docs.moralis.io/introduction/readme)
 
-Try the `<NFTBalance />` component in the [interactive documentation](http://localhost:6006/?path=/docs/1-web3-nft-balance--default)
+Try the `<NFTBalance />` component in the [interactive documentation](http://localhost:6006/?path=/docs/1-web3-parse-nft-balance--default)
```
Comment (AbhinavMV) [inline packages/web3/README.md:104]: This link points to localhost right now...

Could you also update this link and make it similar to other components?

File: packages/web3/README.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ If you want to use this component with the connected server but without adding a
 
 The ConnectButton component automatically adds to the local storage info about the connector user used and will automatically call enableWeb3() after rereshing the page. So if user was connected once it will automatically initialize web3 connection(No need anymore to add UseEffect hook for enableWeb3() after refrshing the page)
```
Comment (AbhinavMV) [inline packages/web3/README.md:83]: Could you fix the spelling of `refreshing`... it's incorrect in two places in this line.

**Review by AbhinavMV — CHANGES_REQUESTED** [review]

**Review by BillyG83 — CHANGES_REQUESTED** [review]
this looks great @nikita12c thanks for progressing this task.... one minor problem is this will break all the links in the readme files 🙈

File: packages/web3/README.md (hunk 1 above)
Comment (nikita12c) [reply packages/web3/README.md:104]: okay I do it !

**Review by nikita12c — COMMENTED** [review]
done with the changes

File: packages/web3/README.md (hunk 2 above)
Comment (nikita12c) [reply packages/web3/README.md:83]: done

File: packages/web3/README.md (hunk 1 above)
Comment (nikita12c) [reply packages/web3/README.md]: done

**Comment by AbhinavMV** [comment]
> this looks great @nikita12c thanks for progressing this task.... one minor problem is this will break all the links in the readme files 🙈
@BillyG83 
I think @nikita12c has fixed all links in the readme files
Maybe I missed something 🤔  or are you talking about links in other moralis docs ?? That might be a problem but since we are going to deprecate servers and end support for this package maybe we can ignore it?

**Review by BillyG83 — APPROVED** [review]
nice work @nikita12c thanks for this update

**Pull request merged**